code/modules/recycling/conveyor.dm

```diff
@@ -10,21 +17,37 @@ GLOBAL_LIST_EMPTY(conveyors_by_id)
 	layer = BELOW_OPEN_DOOR_LAYER
 	processing_flags = START_PROCESSING_MANUALLY
 	subsystem_type = /datum/controller/subsystem/processing/conveyors
-	var/operating = 0 // 1 if running forward, -1 if backwards, 0 if off
-	var/operable = 1 // true if can operate (no broken segments in this belt run)
-	var/forwards // this is the default (forward) direction, set by the map dir
-	var/backwards // hopefully self-explanatory
-	var/movedir // the actual direction to move stuff in
-	var/id = "" // the control ID - must match controller ID
-	var/verted = 1 // Inverts the direction the conveyor belt moves.
+	/// The current operating status. CONVEYOR_OFF (0) if off, CONVEYOR_FORWARD (1) if running forward, CONVEYOR_BACKWARDS (-1) if backwards.
```

Comment:
why include the values?

Reply:
Removed, I reworded it in a way that should hopefully be clearer.